diff -burN libjpeg-turbo-1.4.1.orig/jmorecfg.h libjpeg-turbo-1.4.1/jmorecfg.h
--- libjpeg-turbo-1.4.1.orig/jmorecfg.h	2015-07-10 13:56:02.001667000 +0200
+++ libjpeg-turbo-1.4.1/jmorecfg.h	2015-07-10 13:58:10.002120000 +0200
@@ -204,6 +204,9 @@
  * specific header files that you want to include together with these files.
  * Defining HAVE_BOOLEAN before including jpeglib.h should make it work.
  */
+#if defined(__RPCNDR_H__)
+#define HAVE_BOOLEAN
+#endif
 
 #ifndef HAVE_BOOLEAN
 typedef int boolean;
