diff -burN libjpeg-turbo-1.4.1/jmorecfg.h libjpeg-turbo-1.4.1.orig/jmorecfg.h
--- libjpeg-turbo-1.4.1/jmorecfg.h	2014-11-25 11:07:43.000000000 +0100
+++ libjpeg-turbo-1.4.1.orig/jmorecfg.h	2015-07-10 13:19:46.000911000 +0200
@@ -103,6 +103,11 @@
 #else /* not HAVE_UNSIGNED_CHAR */
 
 typedef char JOCTET;
+
+#ifdef _WIN32
+#include <basetsd.h>
+#endif
+
 #ifdef __CHAR_UNSIGNED__
 #define GETJOCTET(value)  (value)
 #else
@@ -121,6 +126,8 @@
 
 /* UINT8 must hold at least the values 0..255. */
 
+/* Microsoft and MinGW defines it in basetsd.h */
+#if !defined(_BASETSD_H_) && !defined(_BASETSD_H)
 #ifdef HAVE_UNSIGNED_CHAR
 typedef unsigned char UINT8;
 #else /* not HAVE_UNSIGNED_CHAR */
@@ -130,21 +137,25 @@
 typedef short UINT8;
 #endif /* __CHAR_UNSIGNED__ */
 #endif /* HAVE_UNSIGNED_CHAR */
-
+#endif /* not _BASETSD_H */
 /* UINT16 must hold at least the values 0..65535. */
 
+/* Microsoft and MinGW defines it in basetsd.h */
+#if !defined(_BASETSD_H_) && !defined(_BASETSD_H)
 #ifdef HAVE_UNSIGNED_SHORT
 typedef unsigned short UINT16;
 #else /* not HAVE_UNSIGNED_SHORT */
 typedef unsigned int UINT16;
 #endif /* HAVE_UNSIGNED_SHORT */
+#endif
 
 /* INT16 must hold at least the values -32768..32767. */
-
+/* Microsoft and MinGW defines it in basetsd.h */
+#if !defined(_BASETSD_H_) && !defined(_BASETSD_H)
 #ifndef XMD_H                   /* X11/xmd.h correctly defines INT16 */
 typedef short INT16;
 #endif
-
+#endif
 /* INT32 must hold at least signed 32-bit values. */
 
 #ifndef XMD_H                   /* X11/xmd.h correctly defines INT32 */
diff -burN libjpeg-turbo-1.4.1/jpeglib.h libjpeg-turbo-1.4.1.orig/jpeglib.h
--- libjpeg-turbo-1.4.1/jpeglib.h	2014-11-25 11:07:43.000000000 +0100
+++ libjpeg-turbo-1.4.1.orig/jpeglib.h	2015-07-10 13:16:16.000834000 +0200
@@ -35,6 +35,7 @@
 #endif
 #endif
 
+#include <stdio.h>
 
 /* Various constants determining the sizes of things.
  * All of these are specified by the JPEG standard, so don't change them
