diff -burN muparserx_v3_0_3.orig/CMakeLists.txt muparserx_v3_0_3/CMakeLists.txt
--- muparserx_v3_0_3.orig/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ muparserx_v3_0_3/CMakeLists.txt	2015-03-15 17:10:28.159547177 +0100
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 2.8)
+file(GLOB MUPARSERX_SRCS "${CMAKE_SOURCE_DIR}/parser/*.cpp" )
+
+add_library(muparserx ${MUPARSERX_SRCS})
+
+install(TARGETS muparserx
+  RUNTIME DESTINATION bin COMPONENT RuntimeLibraries
+  LIBRARY DESTINATION lib COMPONENT RuntimeLibraries
+  ARCHIVE DESTINATION lib COMPONENT Development)
+
+file(GLOB MUPARSERX_HEADERS "${CMAKE_SOURCE_DIR}/parser/*.h")
+install(FILES ${MUPARSERX_HEADERS}
+  DESTINATION include
+  COMPONENT Development)
+
+#only for muparser 3.0.3 or less
+install(FILES "${CMAKE_SOURCE_DIR}/parser/mpCompat.h"
+  DESTINATION include
+  COMPONENT Development)
+
diff -burN muparserx_v3_0_3.orig/parser/mpCompat.h muparserx_v3_0_3/parser/mpCompat.h
--- muparserx_v3_0_3.orig/parser/mpCompat.h	1970-01-01 01:00:00.000000000 +0100
+++ muparserx_v3_0_3/parser/mpCompat.h	2015-03-15 17:04:00.506196039 +0100
@@ -0,0 +1,55 @@
+/*
+               __________                                 ____  ___
+    _____  __ _\______   \_____ _______  ______ __________\   \/  /
+   /     \|  |  \     ___/\__  \\_  __ \/  ___// __ \_  __ \     /
+  |  Y Y  \  |  /    |     / __ \|  | \/\___ \\  ___/|  | \/     \
+  |__|_|  /____/|____|    (____  /__|  /____  >\___  >__| /___/\  \
+        \/                     \/           \/     \/           \_/
+                                       Copyright (C) 2014 Ingo Berg
+                                       All rights reserved.
+
+  muParserX - A C++ math parser library with array and string support
+  Copyright (c) 2013, Ingo Berg
+  All rights reserved.
+
+  Redistribution and use in source and binary forms, with or without
+  modification, are permitted provided that the following conditions are met:
+
+   * Redistributions of source code must retain the above copyright notice,
+     this list of conditions and the following disclaimer.
+   * Redistributions in binary form must reproduce the above copyright notice,
+     this list of conditions and the following disclaimer in the documentation
+     and/or other materials provided with the distribution.
+
+  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+  IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+  INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+  NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  POSSIBILITY OF SUCH DAMAGE.
+*/
+#ifndef MUP_COMPAT_H
+#define MUP_COMPAT_H
+
+// This file contains code to maintain backwards compatibility with older compilers.
+// Note:    This does not work on windows since the lazy bums at microsoft are unable to assign a macro value properly and
+//          don't give a shit in general:
+//
+//          https ://connect.microsoft.com/VisualStudio/feedback/details/763051/a-value-of-predefined-macro-cplusplus-is-still-199711l
+//
+//          I'll hereby officially announce that i also stopped giving a shit whether this library runs on anything older then VS2013. This
+//          is not a bug, please refrein from reporting it. Seriousely, report it to Microsoft.
+#if !defined(_MSC_VER) && __cplusplus <= 199711L
+  // Warning: If you activate this in a project on windows all hell will break loose and Satan himself will come out of hell to eat
+  //          your soul whilst reciting verses from the Microsoft MFC programming manual.
+  #define override
+  #define unique_ptr auto_ptr
+  #define nullptr NULL
+#endif
+
+#endif
+
diff -burN muparserx_v3_0_3.orig/parser/mpTypes.h muparserx_v3_0_3/parser/mpTypes.h
--- muparserx_v3_0_3.orig/parser/mpTypes.h	2015-03-15 16:29:24.734444856 +0100
+++ muparserx_v3_0_3/parser/mpTypes.h	2015-03-15 16:56:04.549674214 +0100
@@ -50,6 +50,7 @@
 //--- muParserX framework ---------------------------------------------------
 #include "suSortPred.h"  // We need the string utils sorting predicates
 #include "mpDefines.h"
+#include "mpCompat.h"
 #include "mpMatrix.h"
