Les sous-répertoires portablexdr-4.9.1.orig/m4 et portablexdr-4.9.1/m4 sont identiques
diff -buN portablexdr-4.9.1.orig/Makefile.am portablexdr-4.9.1/Makefile.am
--- portablexdr-4.9.1.orig/Makefile.am	2015-04-30 12:51:51.903175904 +0200
+++ portablexdr-4.9.1/Makefile.am	2015-04-30 12:53:10.746372787 +0200
@@ -7,7 +7,7 @@
 	xdr.c xdr_intXX_t.c xdr_rec.c xdr_stdio.c \
 	byteswap.h \
 	rpc/rpc.h rpc/types.h rpc/xdr.h
-libportablexdr_la_CFLAGS = -Wall -Werror -fno-strict-aliasing
+libportablexdr_la_CFLAGS = -Wall -fno-strict-aliasing
 libportablexdr_la_LDFLAGS = @MINGW_EXTRA_LDFLAGS@
 
 # Replacement 'rpcgen'.  Don't call it 'rpcgen' however so that
diff -buN portablexdr-4.9.1.orig/Makefile.in portablexdr-4.9.1/Makefile.in
--- portablexdr-4.9.1.orig/Makefile.in	2015-04-30 12:51:51.903175904 +0200
+++ portablexdr-4.9.1/Makefile.in	2015-04-30 12:52:59.681924203 +0200
@@ -248,7 +248,7 @@
 	byteswap.h \
 	rpc/rpc.h rpc/types.h rpc/xdr.h
 
-libportablexdr_la_CFLAGS = -Wall -Werror -fno-strict-aliasing
+libportablexdr_la_CFLAGS = -Wall -fno-strict-aliasing
 libportablexdr_la_LDFLAGS = @MINGW_EXTRA_LDFLAGS@
 
 # Replacement 'rpcgen'.  Don't call it 'rpcgen' however so that
Les sous-répertoires portablexdr-4.9.1.orig/rpc et portablexdr-4.9.1/rpc sont identiques
